pom.xml

```diff
@@ -240,6 +240,22 @@
             </plugin>
 
             <plugin>
+                <groupId>com.coveo</groupId>
+                <artifactId>fmt-maven-plugin</artifactId>
+                <version>1.5.0</version>
+                <configuration>
```

Comment:
nit: Remove the verbose configuration? It adds a lot of noise, but not much value IMO.